src/makeCalculator.js

```diff
@@ -37,7 +37,38 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+
+  return {
+    result: 0,
+
+    operate(callback, number) {
+      this.result = callback(this.result, number);
+
+      return this;
+    },
```

Comment:
pass just value from operate 

```suggestion
    operate(callback, number) {
      callback(number);

      return this;
    },
```